### ISSUE TITLE
Setting Tree Options Allow Drag

### ISSUE THREAD

**Issue by LeahPike** [500tech/angular2-tree-component#90]
I have tried setting this option via a button press but it doesn't take affect. Is this intended?

```
treeOptions = {
    allowDrag: false
}

enableDragDrop() {
    this.treeOptions.allowDrag = true;
}
```

**Comment by adamkleingit** [comment]
Hey,
currently tree options are not binded to.

As a bypass I suggest to update the reference to the options object (should work):
this.treeOptions = Object.assign({}, this.treeOptions, { allowDrag: true});

**Comment by adamkleingit** [comment]
Hi,
fixed in 2.1.0 - you can now update the options object directly.

**Comment by LeahPike** [comment]
Great, I'll give it a try 👍
